routes/public/index.js

```diff
@@ -5,22 +5,44 @@ const api = require('@api')
 
 router.get('/', async (req, res) => {
   const users = ['@someone', '@someoneelse', '@tacos', '@pizza']
-  res.render('public/index', {
-    page: {
-      title: 'Leaderboard',
-      users: users
+  const challenges = [
```

Comment:
You can pull in real data using `const challenges = await api.challenges.get()`

console log the results, and take a look at what's available for you. When rendering them in liquid you can do a for loop:

`{% for challenge in page.challenges %}`

and access the content you want to display by using `{{ challenge.title }}`